assignments/objects.js

```diff
@@ -18,28 +18,70 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+  "id": 1,
+  "name": 'Mitzi',
+  "email": 'mmelloy0@psu.edu',
+  "gender": 'F'
+}
 
+const kennan = {
+  "id": 2,
+  "name": 'Kennan',
+  "email": 'kdiben1@tinypic.com',
+  "gender": 'M'
+}
 
-// ==== Challenge 2: Reading Object Data ==== 
-// Once your objects are created, log out the following requests from HR into the console:
-
-// Mitzi's name
+const keven = {
+  "id": 3,
+  "name": 'Keven',
+  "email": 'kmummery2@wikimedia.org',
+  "gender": 'M'
+}
 
-// Kennan's ID
+const gannie = {
+  "id": 4,
+  "name": 'Gannie',
+  "email": 'gmartinson3@illinois.edu',
+  "gender": 'M'
+}
 
-// Keven's email
+const antonietta = {
+  "id": 5,
+  "name": 'Antonietta',
+  "email": 'adaine5@samsung.com',
+  "gender": 'F'
+}
 
-// Gannie's name
+// ==== Challenge 2: Reading Object Data ==== 
+// Once your objects are created, log out the following requests from HR into the console:
 
-// Antonietta's Gender
+console.log(mitzi["name"]);
+console.log(kennan["id"]);
+console.log(keven["email"]);
+console.log(gannie["name"]);
+console.log(antonietta['gender']);
```

Comment:
Keep in mind that bracket notation isn't necessary here; you could've used dot notation like this:
```javascript
console.log(keven.email);
```